doc/code_of_conduct_enforcement_manual.rst

```diff
@@ -0,0 +1,60 @@
+
+Possible responses may include:
+ * Taking no further action (if we determine no violation occurred).
+ * A private reprimand from the working group to the individual(s) involved. In this case, the group chair will deliver that reprimand to the individual(s) over email, cc'ing the group.
+ * A public reprimand. In this case, the group chair will deliver that reprimand in the same venue that the violation occurred (i.e. in IRC for an IRC violation; email for an email violation, etc.). The group may choose to publish this message elsewhere for posterity.
+ * An imposed vacation (i.e. asking someone to "take a week off" from a mailing list or IRC). The group chair will communicate this "vacation" to the individual(s). They'll be asked to take this vacation voluntarily, but if they don't agree then a temporary ban may be imposed to enforce this vacation.
+ * A permanent or temporary ban from some or all Zephyr spaces (mailing lists, IRC, etc.). The group will maintain records of all such bans so that they may be reviewed in the future, extended to new Zephyr fora, or otherwise maintained.
+ * A request for a public or private apology. The chair will deliver this request. The group may, if it chooses, attach "strings" to this request: for example, the group may ask a violator to apologize in order to retain his or her membership on a mailing list.
+
+Once a resolution is agreed upon, but before it is enacted, the working group will contact the original reporter and any other affected parties and explain the proposed resolution. The working group will ask if this resolution is acceptable, and must note feedback for the record. However, the working group is not required to act on this feedback.
+Finally the working group will make a report for the Zephyr Governing Board (as well as the TSC team in the event of an ongoing resolution, such as a ban). The working group will never publicly discuss the issue; all public statements will be made by the Zephyr Project Governing Board.
+
+Conflicts of Interest
+*********************
+
+In the event of any conflict of interest a working group member must immediately notify the other members, and recuse themselves if necessary.
+
+*Editor's note: Writing this document posed a unique challenge. Most similar guides are written on the assumption of an in-person event. However, the Zephyr community doesn't exist in one place, and most of the time we're spread out across the world and interact online. This makes trying to define and enforce community standards a different type of challenge. This document is adapted from the Django community, who adopted it from the Ada Initiative template and the PyCon 2013 Procedure for Handling Harassment Incidents, but changed to reflect the nature of our community. It is our expectation that this will be a living document and change as we grow to understand how to meet this challenge and best serve our community and ideals.*
```

Comment:
Not sure we're that unique -- many open source projects have world-wide contributors, no?